### ISSUE TITLE
Animation bug

### ISSUE THREAD

**Issue by tosbaha** [TosinAF/HopperBus-iOS#2]
First of thanks for lovely app with great animation. I think there is a small bug about animation. For example Select Route 903 and select the bottom stop, scroll up and select the top bus stop, animation doesn't happen. This behavior can be seen when we select a new cell where old cell is hidden in the view.

**Comment by TosinAF** [comment]
yeah its an issue as if the cell previously selected is no more in the visible cells. dont have time to fix it now but i tried using the last or first visible cell if the previously selected cell is hidden.

have a go if u want :)

**Comment by tosbaha** [comment]
I am trying to fix it but couldn't do it. I tried to select visible cell by using selectRowAtIndexPath but it didn't work out. Btw, I am the one whom you helped to get this awesome code running. I will email you my current additions that you may find useful :)

**Comment by TosinAF** [comment]
Oh i see, i'll have a look this evening. 
Haha that'd be awesome thanks.
